### DISCUSSION TITLE
Welcome to poster Discussions!

### DISCUSSION THREAD

**Discussion by petersem** [petersem/posterr#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by petersem** [comment]
Development has resumed on Posterr. First step is to update old libraries and fix the Sonarr bug. (Caused by a change in the API for Sonarr v3). So the next release will  be bugfixes and stabilisation.

After this, I will look at some feature requests.

**Comment by olvrteil** [comment]
Hello, I would like to share a developpement idea.
Zidoo players are frequently used for dedicated home theater, where posterr screen could be installed.
Would you mind develop support for Zidoo like Posterr already supports Plex ?

Zidoo are API compatible, i already asked on their forum.
[http://forum.zidoo.tv/index.php?threads/posters-of-playing-movie-on-second-screen-at-room-entry.98857/#post-231409](http://forum.zidoo.tv/index.php?threads/posters-of-playing-movie-on-second-screen-at-room-entry.98857/#post-231409)

Thanks for your work. I'd like to use it at home :)

**Reply by petersem** [reply]
Thanks for the info. Posterr requires a little more info than what the playing api has, like ratings and tmdb id for example. In addition, poster needs to be able to look at the existing movies and shows in order to display on demand titles. 

It's possible I could do something if there was more doco about all apis. Even with full api data, I don't have a device to develop against and test, so this isn't feasible to develop.
